packages/@aws-cdk/aws-eks/README.md

```diff
@@ -714,70 +814,132 @@ resource or if Helm charts depend on each other. You can use
+manifests](#kubernetes-resources) and [Helm charts](#helm-charts) on clusters
+that are not defined as part of your CDK app.
 
-With services account you can provide Kubernetes Pods access to AWS resources.
+First, you'll need to "import" a cluster to your CDK app. To do that, use the
+`eks.Cluster.fromClusterAttributes()` static method:
 
 ```ts
-// add service account
-const sa = cluster.addServiceAccount('MyServiceAccount');
+const cluster = eks.Cluster.fromClusterAttributes(this, 'MyCluster', {
+  clusterName: 'my-cluster-name',
+  kubectlRoleArn: 'arn:aws:iam::1111111:role/iam-role-that-has-masters-access',
+});
+```
 
-const bucket = new Bucket(this, 'Bucket');
-bucket.grantReadWrite(serviceAccount);
+Then, you can use `addManifest` or `addHelmChart` to define resources inside
+your Kubernetes cluster. For example:
 
-const mypod = cluster.addManifest('mypod', {
+```ts
+cluster.addManifest('Test', {
   apiVersion: 'v1',
-  kind: 'Pod',
-  metadata: { name: 'mypod' },
-  spec: {
-    serviceAccountName: sa.serviceAccountName
-    containers: [
-      {
-        name: 'hello',
-        image: 'paulbouwer/hello-kubernetes:1.5',
-        ports: [ { containerPort: 8080 } ],
-
-      }
-    ]
-  }
+  kind: 'ConfigMap',
+  metadata: {
+    name: 'myconfigmap',
+  },
+  data: {
+    Key: 'value',
+    Another: '123454',
+  },
 });
+```
 
-// create the resource after the service account.
-// note that using `sa.serviceAccountName` above **does not** translate into a dependency.
-// this is why an explicit dependency is needed. See https://github.com/aws/aws-cdk/issues/9910 for more details.
-mypod.node.addDependency(sa);
+At the minimum, when importing clusters for `kubectl` management, you will need
+to specify:
 
-// print the IAM role arn for this service account
-new cdk.CfnOutput(this, 'ServiceAccountIamRole', { value: sa.role.roleArn })
-```
\ No newline at end of file
+- `clusterName` - the name of the cluster.
+- `kubectlRoleArn` - the ARN of an IAM role mapped to the `system:masters` RBAC
+  role. If the cluster you are importing was created using the AWS CDK, the
+  CloudFormation stack has an output that includes an IAM role that can be used.
+  Otherwise, you can create an IAM role and map it to `system:masters` manually.
+  The trust policy of this role should include the the
+  `arn:aws::iam::${accountId}:root` principal in order to allow the execution
+  role of the kubectl resource to assume it.
+
+If the cluster is configured with private-only or private and restricted public
+Kubernetes [endpoint access](#endpoint-access), you must also specify:
+
+- `kubectlSecurityGroupId` - the ID of an EC2 security group that is allowed
+  connections to the cluster's control security group. For example, the EKS managed [cluster security group](#cluster-security-group).
+- `kubectlPrivateSubnetIds` - a list of private VPC subnets IDs that will be used
+  to access the Kubernetes endpoint.
+
+## Known Issues and Limitations
```

Comment:
There are quite a few issues users should be aware of, even though there are explicitly mentioned in the relevant section, I also think it would be beneficial to have a clear simple section that emphasizes them.